workflow/rules/manta_run_workflow_tn.smk

```diff
@@ -0,0 +1,48 @@
+__author__ = "Martin Rippin"
+__copyright__ = "Copyright 2021, Martin Rippin"
+__email__ = "martin.rippin@scilifelab.uu.se"
+__license__ = "GPL-3"
+
+
+rule manta_run_workflow_tn:
+    input:
+        bam_t="alignment/merge_bam/{sample}_T.bam",
+        bai_t="alignment/merge_bam/{sample}_T.bam.bai",
+        bam_n="alignment/merge_bam/{sample}_N.bam",
+        bai_n="alignment/merge_bam/{sample}_N.bam.bai",
+        ref=config["reference"]["fasta"],
+        scrpt="cnv_sv/manta/{sample}/runWorkflow.py",
+    output:
+        cand_si_vcf=temp("cnv_sv/manta/{sample}/results/variants/candidateSmallIndels.vcf.gz"),
+        cand_si_tbi=temp("cnv_sv/manta/{sample}/results/variants/candidateSmallIndels.vcf.gz.tbi"),
+        cand_sv_vcf=temp("cnv_sv/manta/{sample}/results/variants/candidateSV.vcf.gz"),
+        cand_sv_tbi=temp("cnv_sv/manta/{sample}/results/variants/candidateSV.vcf.gz.tbi"),
+        dipl_sv_vcf=temp("cnv_sv/manta/{sample}/results/variants/diploidSV.vcf.gz"),
+        dipl_sv_tbi=temp("cnv_sv/manta/{sample}/results/variants/diploidSV.vcf.gz.tbi"),
+        som_sv_vcf="cnv_sv/manta/{sample}/results/variants/somaticSV.vcf.gz",
```

Comment:
we should write down something about when temp should be applied to output values, in most existing rule I believe we have it on all output.

Reply:
Ah ok, might be a good idea to minimize the footprint of our workflows.